applications/asset_tracker/README.rst

```diff
@@ -52,6 +52,49 @@ The `LTE Link Monitor`_ application, implemented as part of `nRF Connect for Des
 
 By default, the asset tracker supports firmware updates through :ref:`lib_aws_fota`.
 
+.. _power_opt:
+
+Power optimization
+==================
+
+The Asset Tracker can run in three power modes that are configured in the Kconfig file of the application.
+These settings are currently only supported on the nRF9160 DK.
+
+.. note::
+   Not all cellular network providers support these modes, and the granted parameters can vary between networks.
+
+Demo mode
+	This is the default setting.
+	In this mode, the device maintains a continuous cellular link.
```

Comment:
To be fair to eDRX and PSM, they also offer a continuous link, in the sense that the connection to the cellular network is maintained.
Without eDRX and PSM, the device can receive data at all times, while it cannot with those two modes enabled. 
Then it's only reachable at certain configurable intervals, or when the device sends data to the network.

Reply:
Added the above information.